tests/tck/features/lookup/TagIndexFullScan.feature

```diff
@@ -125,11 +125,69 @@ Feature: Lookup tag index full scan
+      | id | name      | dependencies | operator info |
+      | 3  | Project   | 4            |               |
+      | 4  | IndexScan | 0            |               |
+      | 0  | Start     |              |               |
+    # a IN b OR c IN d
+    When profiling query:
+      """
+      LOOKUP ON player WHERE player.age IN [40, 25] OR player.name IN ["Kobe Bryant"] YIELD player.age
+      """
+    Then the result should be, in any order:
+      | VertexID        | player.age |
+      | "Dirk Nowitzki" | 40         |
+      | "Joel Embiid"   | 25         |
+      | "Kobe Bryant"   | 40         |
+      | "Kyle Anderson" | 25         |
+    And the execution plan should be:
+      | id | name      | dependencies | operator info |
+      | 3  | Project   | 4            |               |
+      | 4  | IndexScan | 0            |               |
+      | 0  | Start     |              |               |
+    # a IN b AND c
+    When profiling query:
+      """
+      LOOKUP ON player WHERE player.age IN [40, 25] AND player.name == "Kobe Bryant" YIELD player.age
+      """
+    Then the result should be, in any order:
+      | VertexID      | player.age |
+      | "Kobe Bryant" | 40         |
+    And the execution plan should be:
+      | id | name               | dependencies | operator info                                                                                                           |
+      | 3  | Project            | 4            |                                                                                                                         |
+      | 4  | TagIndexPrefixScan | 0            | {"indexCtx": {"columnHints":{"endValue":"__EMPTY__","beginValue":"\"Kobe Bryant","scanType":"PREFIX","column":"name"}}} |
+      | 0  | Start              |              |                                                                                                                         |
+    # a IN b AND c IN d (TagIndexFullScan)
+    When profiling query:
+      """
+      LOOKUP ON player WHERE player.age IN [40, 25] AND player.name IN ["ABC", "Kobe Bryant"] YIELD player.age
+      """
+    Then the result should be, in any order:
+      | VertexID      | player.age |
+      | "Kobe Bryant" | 40         |
+    And the execution plan should be:
+      | id | name             | dependencies | operator info                                                                                                            |
+      | 3  | Project          | 2            |                                                                                                                          |
+      | 2  | Filter           | 4            | {"condition": "(((player.age==40) OR (player.age==25)) AND ((player.name==\"ABC\") OR (player.name==\"Kobe Bryant\")))"} |
+      | 4  | TagIndexFullScan | 0            |                                                                                                                          |
```

Comment:
What's the actual operator info of TagIndexFullScan? Could you paste it here?

Reply:
`TagIndexFullScan` scans the whole tag, so the `"columnHints"` is empty.
```
|  4 | TagIndexFullScan | 0            | ver: 0, rows: 56, execTime: 0us, totalTime: 9707us | outputVar: [                                                                                                   |
|    |                  |              |                                                    |   {                                                                                                            |
|    |                  |              |                                                    |     "colNames": [                                                                                              |
|    |                  |              |                                                    |       "VertexID",                                                                                              |
|    |                  |              |                                                    |       "player.age",                                                                                            |
|    |                  |              |                                                    |       "player.name"                                                                                            |
|    |                  |              |                                                    |     ],                                                                                                         |
|    |                  |              |                                                    |     "type": "DATASET",                                                                                         |
|    |                  |              |                                                    |     "name": "__TagIndexFullScan_1"                                                                             |
|    |                  |              |                                                    |   }                                                                                                            |
|    |                  |              |                                                    | ]                                                                                                              |
|    |                  |              |                                                    | inputVar:                                                                                                      |
|    |                  |              |                                                    | space: 1                                                                                                       |
|    |                  |              |                                                    | dedup: false                                                                                                   |
|    |                  |              |                                                    | limit: 9223372036854775807                                                                                     |
|    |                  |              |                                                    | filter:                                                                                                        |
|    |                  |              |                                                    | orderBy: []                                                                                                    |
|    |                  |              |                                                    | schemaId: 2                                                                                                    |
|    |                  |              |                                                    | isEdge: false                                                                                                  |
|    |                  |              |                                                    | returnCols: [                                                                                                  |
|    |                  |              |                                                    |   "_vid",                                                                                                      |
|    |                  |              |                                                    |   "age",                                                                                                       |
|    |                  |              |                                                    |   "name"                                                                                                       |
|    |                  |              |                                                    | ]                                                                                                              |
|    |                  |              |                                                    | indexCtx: [                                                                                                    |
|    |                  |              |                                                    |   {                                                                                                            |
|    |                  |              |                                                    |     "columnHints": [],                                                                                         |
|    |                  |              |                                                    |     "filter": "",                                                                                              |
|    |                  |              |                                                    |     "index_id": 8                                                                                              |
|    |                  |              |                                                    |   }                                                                                                            |
|    |                  |              |                                                    | ]                                                                                                              |
```

Reply:
So why does the condition of `filter` not be pushed (or be partitially pushed) into `IndexScan`?